src/ahbicht/validation/validation.py

```diff
@@ -330,6 +330,14 @@ async def validate_data_element_valuepool(
     if not possible_values:
         requirement_validation_data_element is RequirementValidationValue.IS_FORBIDDEN
         hints = None
+    else:
+        if data_element.entered_input in possible_values:
+            requirement_validation_data_element = RequirementValidationValue.IS_REQUIRED_AND_FILLED
+        elif data_element.entered_input:
+            requirement_validation_data_element = RequirementValidationValue.IS_REQUIRED
+            hints = f"Der Wert '{data_element.entered_input}' ist nicht in: {{{', '.join(possible_values.keys())}}}"
```

Comment:
der fall: user gibt wert ein, der nicht teil des wertevorrats ist

Reply:
Hm, wir sind davon ausgegangen, dass man gar keinen Wert auswählen kann, der nicht erlaubt ist, weil das FE ja das Dropdown aus dem possible_values generiert

Reply:
In Absprache mit Franzi ist die Überlegung den Value wieder auf EMTPY zu setzten und die requirement validation auf IS_REQUIRED_AND_EMPTY